### DISCUSSION TITLE
How do you step-by-step list the speaker indexes?

### DISCUSSION THREAD

**Discussion by shyamalschandra** [coqui-ai/TTS#1214] [General Q&A]
```
tts --model_name "<language>/<dataset>/<model_name>"  --list_speaker_idxs
```

Can you show a real version of this command line program running with a model name selected from a list of models?

**Comment by Ca-ressemble-a-du-fake** [comment]
Hi, I don't have a computer in front of me so the commands may be erroneous but if I remember correctly this is what I did : 

Show the help tts --help or tts -h. There should be a flag to list models.

Tts --list_models

Then you pick one of the model strings that. appear.

Then you can do tts theModel/you/picked/exactluAsItAppeared/Above --list_languages

Then you add the language flag and you can do
Tts  theModel/you/picked/exactluAsItAppeared/Above "fr-fr" --list_speaker_idxs

And you'll have the speaker name you can use in the full command 
Tts  theModel/you/picked/exactluAsItAppeared/Above --list_languages "fr-fr" "speakerName"

**Comment by wangtiance** [comment]
tts --model_name tts_models/en/vctk/vits --list_speaker_idxs

list_speaker_idxs only works with models trained with a multi-speaker dataset, like vctk
